Jenkinsfile

```diff
@@ -7,44 +7,31 @@ pipeline {
     }
 
     stages {
-        stage('Build') {
+        stage('Commit & Upload to Nexus') {
```

Comment:
I propose standardizing this stage name to "Commit" in the future for all CI pipelines!